Templates/GitHubWebHook-CSharp/run.csx

```diff
@@ -0,0 +1,15 @@
+using System.Net;
+using System.Threading.Tasks;
+
+public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter log)
+{
+    log.Verbose($"C# HTTP trigger function processed a request. RequestUri={req.RequestUri}");
+
+    // Get request body
+    dynamic data = await req.Content.ReadAsAsync<object>();
```

Comment:
Do you mean something like this?  
```
JObject data = await req.Content.ReadAsAsync<JObject>();
string gitHubComment = data?["comment"]?["body"].ToString();
```
From what i see in the Script SDK the httprequest is passed on to the function from WebHookHandlerContext